entries/k/khanacademy.org.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Khan Academy": {
+    "domain": "khanacademy.org",
+    "contact": {
+      "twitter": "khanacademy",
+      "facebook": "khanacademy"
```

Comment:
Something weird is going on with that Facebook page as seems behave more like a Facebook profile or fan page rather than a normal page.

Take CIBC as an example:
![CIBC](https://user-images.githubusercontent.com/3535780/201805451-080eaa91-3d13-4b46-8cb9-4a50393521a0.png)
Here the page we can see the handle and category. There's also an option to contact CIBC through Messanger.

![Khan Academy](https://user-images.githubusercontent.com/3535780/201805765-bb19afea-0f75-41bb-8196-5d9866097048.png)
On KA's page the handle and category is replaced with a followers/following count and there's no way to contact the org.

Due to the inconsistencies and since there doesn't seem to be any way to contact KA through the page, I recommend you remove the listing.
```suggestion
```

Reply:
Thanks!